.github/CODEOWNERS

```diff
@@ -1,5 +1,7 @@
 * @microsoft/prompt-flow-approvers
 
+/src/promptflow-tracing/ @microsoft/promptflow-tracing
```

Comment:
I think we don't have a team called promptflow-tracing?

Reply:
actually, i created one: <https://github.com/orgs/microsoft/teams/promptflow-tracing/members>; please let me know if we need another name...

Reply:
mmm... l originally thinks we should keep using the executor and sdk one. We should have an offline sync with Heyi on this.

Reply:
reverted related commit to not block this pr